src/components/channel/SubscribeButton.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { action } from '@kadira/storybook'
+
+export const SubscribeButton = ({
+  children,
+  className = '',
+  channel,
+  showSubscribers,
+  ...rest
+}) => (
+  <div className={`subscribe-button ${className}`} {...rest}>
+    {showSubscribers &&
+      <div className='subscriber-count'>{channel.subscriber_count} subscribers</div>
```

Comment:
`subscriber_count` will be camelized as `subscriberCount` when it comes from the server.